src/pages/APSSocietyPage.js

```diff
@@ -123,8 +123,10 @@ export default function APSSocietyPage(props) {
               </Grid>
             ))}
           </Grid>
-        </Paper>
-        <AlumniAccordions members={alumni.aps} color="rgb(110 110 193)" />
+        </Paper> */}
+
+           <SocietyExec sid = {ecats.aps}/>
+        <AlumniAccordions members={alumni.aps} sid={ecats.aps} color="rgb(110 110 193)" />
```

Comment:
why are we passing the member as props where it is being fetched in the `AlumniAccordions` Component?